NEWS.md

```diff
@@ -73,6 +73,33 @@ unit = "s")
 
 10. The dimensions of objects in a `list` column are now displayed, [#3671](https://github.com/Rdatatable/data.table/issues/3671). Thanks to @randomgambit for the request, and Tyson Barrett for the PR.
 
+11. New function `topn`, implemented in C by Morgan Jacob, [#3804](https://github.com/Rdatatable/data.table/issues/3804). It returns the top largest or smallest `n` values for a given numeric vector `vec`. Please see `?topn` for more details. Similar to `dplyr::top_n`.
```

Comment:
Clarified here... actually not inspired by `top_n`, it was inspired by Matt's comment on https://github.com/Rdatatable/data.table/pull/3604. Note the NEWS item there. Then in writing the issue around that, I noticed `dplyr` already had their version :)